console/migrations/m161218_144901_create_person_table.php

```diff
@@ -1,31 +0,0 @@
-<?php
-
```

Comment:
why did you delete this file? If you wanted to show only the certain hometask files, use different branches and create the pull request between them